templates/function-base/ts/default/.env.teamsfx.local

```diff
@@ -0,0 +1,10 @@
+# Following variables are generated by TeamsFx
```

Comment:
Use the updated comment "# TeamsFx will overwrite the following variable values when running debug"

Reply:
Good catch. Thx!